docs/remote/ssh.md

```diff
@@ -111,7 +111,7 @@ If you are using a Linux or macOS SSH host, you can use the Remote - SSH and [Re
 To do so:
 
 1. Follow the [installation](/docs/remote/containers.md#installation) steps for the Remote - Containers extension on your remote host.
-1. **[Optional]** Set up SSH [key based authentication](/docs/remote/troubleshooting.md#configuring-key-based-authentication) to the server so you do not need to enter your password multiple times.
+1. **\[Optional]** Set up SSH [key based authentication](/docs/remote/troubleshooting.md#configuring-key-based-authentication) to the server so you do not need to enter your password multiple times.
```

Comment:
These seem fine on the code.visualstudio.com.
If they are confusing in Markdown, we could just change them to simple bold text.
Maybe just **Optional:** using a colon to separate them from the following text instead of brackets